### ISSUE TITLE
line-wrap long items?

### ISSUE THREAD

**Issue by Gankra** [mozilla/cargo-vet#176]
Notable candidates:

* dependency-criteria (inline array gets crammed all on one line)
* notes (currently no way to input multiple lines in interactive prompt, creates very long lines)

**Comment by bholley** [comment]
Yeah. Perhaps we should just detect/estimate the number of characters in the inline table and automatically split it out into a subtable if it's more than, say, 120?

**Comment by mystor** [comment]
That's easy enough to do, sounds good.
